packages/blip/app/components/chart/daily.js

```diff
@@ -170,11 +177,18 @@ class DailyChart extends React.Component {
   }
 
   handleWindowResize = ({windowHeight: height, windowWidth: width}) => {
-    const { windowHeight, windowWidth } = this.state;
+    const { loading } = this.props;
+    const { windowHeight, windowWidth, chart } = this.state;
     this.log.debug('handleWindowResize', { windowHeight, windowWidth }, '=>', { height, width });
     if (windowHeight !== height || width !== windowWidth) {
-      this.setState({ windowHeight: height, windowWidth: width }, () => {
-        this.reCreateChart();
+      const needRecreate = loading || chart?.isInTransition() === true;
+      this.setState({ windowHeight: height, windowWidth: width, needRecreate }, () => {
+        this.props.trackMetric('daily', { event: 'resize', width, height });
```

Comment:
wondering why we are tracking this type of event
That would make sense if we had the original size, otherwise we won't have enough information to use this event.

Reply:
When the graph is first displayed, the event is triggered. So we will have all the changes here